openedx/adg/lms/webinars/admin.py

```diff
@@ -36,10 +38,10 @@ def lookups(self, request, model_admin):
 
     def queryset(self, request, queryset):
         if self.value() == Webinar.UPCOMING:
-            return queryset.filter(status=Webinar.UPCOMING)
+            return queryset.filter(is_cancelled=False, start_time__gte=now())
```

Comment:
After adding methods in Webinar queryset, we can use `queryset.upcoming_webinars()`

Reply:
Same for all usages.